cmake/onnxruntime_unittests.cmake

```diff
@@ -377,27 +377,25 @@ endif()
 
 # Disable training ops test for minimal build as a lot of these depend on loading an onnx model.
 if (NOT onnxruntime_MINIMAL_BUILD)
-  if (onnxruntime_ENABLE_TRAINING_CORE)
-    file(GLOB_RECURSE orttraining_test_trainingops_cpu_src CONFIGURE_DEPENDS
-      "${ORTTRAINING_SOURCE_DIR}/test/training_ops/compare_provider_test_utils.cc"
-      "${ORTTRAINING_SOURCE_DIR}/test/training_ops/function_op_test_utils.cc"
-      "${ORTTRAINING_SOURCE_DIR}/test/training_ops/cpu/*"
-      )
+  file(GLOB_RECURSE orttraining_test_trainingops_cpu_src CONFIGURE_DEPENDS
```

Comment:
is it better we use onnxruntime_ENABLE_TRAINING_OPS to control the op tests?

Reply:
Will do.
But from what I understand, `onnxruntime_ENABLE_TRAINING_OPS` and `onnxruntime_ENABLE_TRAINING_CORE` should always be on at the same time. (when `--enable_training` flag is provided). Trying to figure out where the discrepancy comes from.

Reply:
Yes, your intention is correct, while onnxruntime_ENABLE_TRAINING_CORE is undefined